openapi.yaml

```diff
@@ -2985,6 +2985,61 @@ paths:
         - lang: CLI
           source: >
             linode-cli domains delete 1234
+  /domains/{domainId}/zone-file:
+    x-linode-cli-command: domains
+    parameters:
+    - name: domainId
+      in: path
+      description: ID of the Domain.
+      required: true
+      schema:
+        type: string
+    get:
+      x-linode-grant: read_only
+      tags:
+      - Domains
+      summary: Domain Zone File View
+      description: >
+        Returns the zone file for the last rendered zone for the specified domain.
+      operationId: getDomainZone
+      x-linode-cli-action: zone-file
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - domains:read_only
+      responses:
+        '200':
+          description: |
+            The domain zone file.
```

Comment:
This isn't returning a file, rather the contents of that file, one line per array index. Changing the language here to specify that would be helpful.